Integrations/integration-PagerDuty.yml

```diff
@@ -0,0 +1,475 @@
+    ON_CALL_BY_SCHEDULE_SUFFIX = "schedules/{0}/users"
+    ON_CALLS_USERS_SUFFIX = "oncalls?include%5B%5D=users"
+    GET_INCIDENTS_SUFFIX = "incidents?include%5B%5D=assignees"
+
+    '''TABLE NAMES'''
+    SCHEDULES = "All Schedules"
+    TRIGGER_EVENT = 'Trigger Event'
+    USERS_ON_CALL = 'Users On Call'
+    USERS_ON_CALL_NOW = 'Users On Call Now'
+
+    '''TABLE HEADERS'''
+    SCHEDULES_HEADERS = ["ID", "Name", "Today", "Time Zone", "Escalation Policy", "Escalation Policy ID"]
+    USERS_ON_CALL_NOW_HEADERS = ["ID", "Email", "Name", "Role", "Color", "User Url", "Avatar Url", "Time Zone"]
+    INCIDENTS_HEADERS = ["ID", "Incident Number", "Created On", "Status", "Html Url", "Service ID", "Service Name", "Service Html Url", "Assigned To User",
+                         "Escalation Policy", "Escalation Policy ID", "Trigger Details Html Url", "Trigger Type", "Last Status Change On", "Last Status Change By",
+                         "Number Of Escalations", "Resolved By User", "Assigned To", "Summary"]
+
+    ''' HELPER FUNCTIONS '''
+    def http_request(method, url_suffix, params_dict):
+        url = SERVER_URL + url_suffix
+
+        LOG('running %s request with url=%s\theaders=%s\nparams=%s' % (method, url, DEFAULT_HEADERS, json.dumps(params_dict)))
+
+        try:
+            res = requests.request(method,
+                                   url,
+                                   verify=USE_SSL,
+                                   params=params_dict,
+                                   headers=DEFAULT_HEADERS,
+                                  )
+            res.raise_for_status()
+
+            return res.json()
+
+        except Exception, e:
+            LOG(e)
+            raise(e)
+
+
+    def test_module():
+        try:
+            get_users_on_call_now_command()
+        except Exception, e:
+            raise Exception(e.message)
+
+        demisto.results("ok")
+
+    def extract_on_call_user_data(users):
+        """Extact data about user from a given schedule."""
+        outputs = []
+        context = []
+        for i, user in enumerate(users):
+            output = {}
+
+            output['ID'] = user.get('id')
+            output['Name'] = user.get('name')
+            output['Role'] = user.get('role')
+            output['Color'] = user.get('color')
+            output['Email'] = user.get('email')
+            output['User Url'] = user.get('html_url')
+            output['Time Zone'] = user.get('time_zone')
+            output['Avatar Url'] = user.get('avatar_url')
+
+            outputs.append(output)
+            context.append(output)
+
+        return {
+            'Type': entryTypes['note'],
+            'Contents': outputs,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown(USERS_ON_CALL, outputs, USERS_ON_CALL_NOW_HEADERS),
+            'EntryContext': {
+                'PagerDutyUser': context
```

Comment:
@ronykoz did you make sure that the context is the same as in the old integration? Please run the commands one time in the old and one time in the new and make sure it's the same, let's make sure we are backwards compatible.